test/functional/issue_assignments_controller_test.rb

```diff
@@ -0,0 +1,35 @@
+require 'test_helper'
+
+class IssueAssignmentsControllerTest < ActionController::TestCase
+  include Devise::Test::ControllerHelpers
+
+  fixtures :users, :repo_subscriptions, :issue_assignments
```

Comment:
We don't need to load individual fixtures as all fixtures are loaded by https://github.com/codetriage/codetriage/blob/af1de44c5075546e9b5d6ce2cff166fe678bbe99/test/test_helper.rb#L19

Reply:
done!